src/style-spec/validate_style.min.js

```diff
@@ -26,8 +26,7 @@ import validateLayoutProperty from './validate/validate_layout_property';
  *   var validate = require('mapbox-gl-style-spec/lib/validate_style.min');
  *   var errors = validate(style);
  */
-function validateStyleMin(style, styleSpec) {
-    styleSpec = styleSpec || latestStyleSpec;
+function validateStyleMin(style, styleSpec = latestStyleSpec) {
```

Comment:
In the preceding file ([link](https://github.com/mapbox/mapbox-gl-js/pull/8663/files#diff-d51d6c6dc5a36091136162e69a7323d4R34)), `options` (which allows setting `mapboxApiSupported`) gets passed to this function, but I don't see it in the signature of this function. Should it be here?

Reply:
Oh, maybe the change in `validate_style.js` was a holdover from a previous approach at combining the two validation modes?

Reply:
Oh yeah good catch – I need to remove that